scripts/rollup/lwcFeatures.js

```diff
@@ -14,8 +14,13 @@ module.exports = function lwcFeatures() {
     return {
         id: 'rollup-plugin-lwc-features',
         transform(source, id) {
-            if (id.includes('/node_modules/') || !source.includes('@lwc/features')) {
+            if (
+                id.includes('/node_modules/') ||
+                id.includes('/dist/') ||
+                !source.includes('@lwc/features')
+            ) {
```

Comment:
This was the key to making the feature flags work. `@lwc/engine-dom` was pulling in the `dist` file from `@lwc/engine-core`, which had already been processed by the features Babel plugin. So the features Babel plugin was complaining that only the default export should be used.